app/models/form_profile.rb

```diff
@@ -223,43 +223,15 @@ def initialize_military_information_vaprofile
     end
   end
 
-  # When military_information_vaprofile Flipper is removed, this method will be shorter.
-  # rubocop:disable Metrics/MethodLength
   def initialize_military_information
     return {} unless user.authorize :va_profile, :access?
 
-    if Flipper.enabled?(:military_information_vaprofile)
-      military_information_data = {}
-      military_information_data.merge!(initialize_va_profile_prefill_military_information)
-      military_information_data[:vic_verified] = user.can_access_id_card?
-
-    else
-      military_information = user.military_information
-      military_information_data = {}
-
-      military_information_data.merge!(initialize_military_information_vaprofile)
```

Comment:
Double check me, but I think this line removal means you can remove the method on line 207 and then all of `lib/hca/military_information.rb` and those tests.

Reply:
Absolutely right! `initialize_military_information_vaprofile` is now deprecated. I removed all references to this method and the file/specs mentioned above. LMK if this looks g2g now!